### PR TITLE
docker-registry: fix a typo

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: incubator/docker-registry/Chart.yaml [hunk 1]
```diff
@@ -8,3 +8,5 @@ keywords:
 maintainers:
 - name: sbezverk
   email: sbezverk@cisco.com
+- name: Andrey Arapov
+  email: andrey.arapov@nixaid.com
```
Comment (unguiculus) [inline incubator/docker-registry/Chart.yaml:12]: Can you clarify why you are adding yourself as a maintainer?

Comment (ghost) [reply incubator/docker-registry/Chart.yaml:12]: I have a wish to be. :-)
Would this be possible?

Comment (unguiculus) [reply incubator/docker-registry/Chart.yaml:12]: OK, I'll leave you in. You will have to review future changes.

**Pull request merged**